src/services/MpdfProvider.php

```diff
@@ -45,6 +45,7 @@ public function getInstance(): Mpdf
                 ),
             ),
             'default_font' => 'lato',
+            'whitelistStreamWrappers' => array('file'),
```

Comment:
Override https://github.com/mpdf/mpdf/blob/v8.0.17/src/Config/ConfigVariables.php#L299 . Follow usage eg from `ImageProcessor::getImage` (or `AssetFetcher::fetchDataFromPath`, as they seem to have changed it to in their devel branch).